pages/apim/3.x/installation-guide/upgrades/3.18.0/README.adoc

```diff
@@ -36,3 +36,20 @@ To provide more flexibility in the way roles are managed, some system roles have
 . The Application Primary Owner Role
 
 The Organization Admin Role remains a read-only role. If any accidental loss of access happened to one of the roles listed above, the Organization Admin Role will be the only one able to revert the changes.
+
+=== Plugin renaming
+
+From this version, and for the next 3.18.x versions and greater, the name of the Elasticsearch repository component changes. +
+As a consequence, the Elasticsearch repository component available on https://download.gravitee.io is now +
+
+`gravitee-*apim*-repository-elasticsearch-x.y.z.zip` +
+
+instead of +
```

Comment:
The `+` is weird no?

Reply:
It is an Asciidoc syntax to force a `\n`.
But yes, it is useless, since I explicitly put a new line just after 😛